include/openthread-config-android.h

```diff
@@ -26,6 +26,15 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* Define to 1 to enable Backbone Router support. */
+#define OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE 1
+
+/* Define to 1 to enable Backbone Router Multicast Routing for Thread 1.2 MLR feature. */
+#define OPENTHREAD_CONFIG_BACKBONE_ROUTER_MULTICAST_ROUTING_ENABLE 1
+
+/* Define to 0 to disable Backbone Router ND Proxying for Thread 1.2 DUA feature. */
+#define OPENTHREAD_CONFIG_BACKBONE_ROUTER_DUA_NDPROXYING_ENABLE 0
+
```

Comment:
should `OPENTHREAD_CONFIG_TMF_PROXY_DUA_ENABLE` and `OPENTHREAD_CONFIG_TMF_PROXY_MLR_ENABLE` be defined explicitly as well? or maybe plus the other low power relative flags?

Thoughts?